### PR TITLE
Packages/py sphinx removed in

### PR DESCRIPTION
**Pull request by Bidibulke**
Adding recipe for `sphinx-removed-in` (a python package)

### CODE REVIEW

**Review by tldahlgren — CHANGES_REQUESTED** [review]
Please change the Copyright year for both files from 2019 to 2020.

**Comment by Bidibulke** [comment]
copyright changed as requested

File: var/spack/repos/builtin/packages/py-sphinx-removed-in/package.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    .. versionremoved:: and .. removed-in directives.
+    These are missing from Sphinx'es core markup.
+    ."""
+
+    homepage = "https://github.com/MrSenko/sphinx-removed-in"
+    url      = "https://pypi.io/packages/source/s/sphinx-removed-in/sphinx-removed-in-0.2.0.tar.gz"
+
+    version('0.2.0', sha256='bdba7f212c1abd99a9e9c2144c75a41edca02e71dfbb77653be39abe2cb19087')
+
+    depends_on('py-sphinx')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-sphinx-removed-in/package.py]:
```suggestion
    depends_on('py-setuptools', type='build')
    depends_on('py-sphinx', type=('build', 'run'))
```

**Review by adamjstewart — CHANGES_REQUESTED** [review]

File: var/spack/repos/builtin/packages/py-sphinx-removed-in/package.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PySphinxRemovedIn(PythonPackage):
+    """This is a Sphinx extension which recognizes the
+    .. versionremoved:: and .. removed-in directives.
+    These are missing from Sphinx'es core markup.
+    ."""
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-sphinx-removed-in/package.py]:
```suggestion
    """
```

**Review by adamjstewart — APPROVED** [review]

**Comment by alalazo** [comment]
Closing as stale, but feel free to rebase and reopen (in the meanwhile the copyright year has changed to 2021 :grimacing: )

**Pull request closed without merging**